gulpfile.js/tasks/comp-lib.js

```diff
@@ -1,30 +1,22 @@
 'use strict'
 
-var del = require('del')
-var path = require('path')
-var gulp = require('gulp')
-var exec = require('child_process').exec
-var config = require('../config')
-var compLibConfig = require('../../component-lib.json')
+const path = require('path')
+const exec = require('child_process').exec
+const gulp = require('gulp')
+const gutil = require('gulp-util')
+const config = require('../config')
+const compLibConfig = require('../../component-lib.json')
 
-gulp.task('clean-comp-lib', function (cb) {
-  del(compLibConfig.destination, cb)
-})
-
-gulp.task('component-library', ['clean-comp-lib', 'sass', 'images', 'browserify'], function (cb) {
-  var env = global.runmode
-  var genCompLib = './node_modules/.bin/kss-node --config component-lib.json'
-
-  exec(genCompLib, function (err, stout, sterr) {
-    var files = [
-      config.images[env].dest + '/**/*',
-      config.sass[env].dest + '**/*',
-      config.scripts[env].dest + '/**/*'
-    ]
+// @FIXME: should this be a step of build:v3?
```

Comment:
The component library depends on the built artefacts of `build:v3` (so it would make to me to make that job a dependency of this one) but running `build:v3` on its own shouldn't generate the component library.